api.jsx

```diff
@@ -16,4 +16,27 @@ export const fetchArticles = async (topic) => {
   }
 };
 
-export default { fetchArticles };
+export const fetchArticleById = async (articleId) => {
+  try {
+    const response = await ncNewsAPI.get(`/articles/${articleId}`);
+    console.log("GET request successful, response:", response);
+    return response;
+  } catch (error) {
+    console.error("Error status: ", error.response.status);
+    console.error("Error data: ", error.response.data);
+    return await Promise.reject(error);
+  }
+};
+
+export default { fetchArticles, fetchArticleById };
+
+//Endpoints--------//
+
+// app.get("/api/topics", getAllTopics);
```

Comment:
These comments shouldn't be here in this PR.

Reply:
OK, deleted.